### ISSUE TITLE
Minimal footer accessibility updates

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/vets-design-system-documentation#2274]
## Description

A11y issues found in review.

## Tasks
- **Must:** Image link should have the yellow focus outline.
<img width="1299" alt="image" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/100248909/42daa46f-bd40-46b5-bd7e-fab5b0c3cb64">

## Acceptance Criteria
- [ ] When tabbing to the element, the VA yellow focus indicator is shown.

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2274?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by jamigibbs** [comment]
There is a [mixin](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/mixins/focusable.css) that can be used for this. Here is an example of how it's imported into va-accordion:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-accordion/va-accordion.scss#L8
